src/analysis/special_functions/arsinh.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2020 James Arthur. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Arthur, Chris Hughes, Shing Tak Lam.
+-/
+import analysis.special_functions.trigonometric
+import data.real.basic
+import analysis.special_functions.pow
+
+open real
+local attribute [pp_nodot] real.log
```

Comment:
Perhaps this could even be a global instance? Together with a similar instance for `sqrt`?

Reply:
I've already marked `sqrt` as `@[pp_nodot]` in an earlier PR.

@jamesa9283 What you need to do is change https://github.com/leanprover-community/mathlib/blob/cd1c00d54e4c84a58b80868229d6cb4c7baa58cd/src/analysis/special_functions/exp_log.lean#L201 to say 

```lean
@[pp_nodot] noncomputable def log (x : ℝ) : ℝ :=
```

instead, and delete the local attribute line here.

Reply:
Done!